config/kubernetes/sig-testing/teams.yaml

```diff
@@ -62,16 +62,13 @@ teams:
   test-infra-admins:
     description: admin access to test-infra
     members:
+    - alvaroaleman
+    - aojea
     - BenTheElder
-    - chaodaiG
-    - chases2
```

Comment:
I'd like to leave the folks who are still oncall (chases2, listx, mpherman2)

But aside from that, LGTM!

Reply:
@michelle192837 then shouldn't we also have the `sig-k8s-infra-oncall` as well?

https://github.com/kubernetes/test-infra/blob/master/OWNERS_ALIASES#L16C3-L22

Reply:
That's fine by me 👍